OpenRA.Mods.Common/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -14,6 +14,7 @@
 using System.Linq;
 using OpenRA.Graphics;
 using OpenRA.Mods.Common.Effects;
+using OpenRA.Mods.Common.Traits;
```

Comment:
This isn't used, probably a left-over from earlier revisions, so it should be removed.